docs/book/promotions.rst

```diff
@@ -4,37 +4,126 @@
 Promotions
 ==========
 
-...
+The system of **Promotions** in **Sylius** is really flexible. It is a combination of promotion rules and actions.
+
+Promotions are assigned to Orders. They have a few parameters - a unique ``code``, ``name``, ``usageLimit``,
```

Comment:
This is confusing, I would remove "Promotions are assigned to Orders.". We should explain that separately when describing the "applying" logic.